### ISSUE TITLE
why input data need to minus 0.4XX

### ISSUE THREAD

**Issue by CXMANDTXW** [twhui/LiteFlowNet#44]
Hello, When I use the pytorch version of the code，I found a similar operation for flownet：

input_transform = transforms.Compose([
transforms.Normalize(mean=[0.411,0.432,0.45], std=[1,1,1])
])

I guess LiteFlowNet maybe have the same operation.
what its effect on the quality of the estimated flow is?
I‘m sorry for my childish question,but I'm very confused about that.

**Comment by twhui** [comment]
Hello! This is normalization trick used by FlowNet. The RGB mean values are computed on-the-fly during  training stage for several iterations. After that, the RGB mean values will not be updated and will be used for subsequent training or fine-tuning. LiteFlowNet-pytorch simply copies the RGB mean values from the pre-trained caffe models.

**Comment by twhui** [comment]
closed due to in inactivity

**Comment by abrosua** [comment]
Hi, I want to ask about where to find the the RGB mean value from the train prototxt.

Since I want to recreate the PyTorch version of this [project](https://github.com/shengzesnail/PIV-LiteFlowNet-en) that used your LiteFlowNet model trained on a specific type of datasets, perhaps they produced a different RGB value?
